### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bcpswope**
**Description of the change:** We need to change the link to prod. We need to use the product name officially.

@dhaiducek and I are wondering how external users get to an internal channel?

**Motivation for the change:** Product guidelines and repo structure.

### CODE REVIEW

**Comment by tpouyer** [comment]
> **Description of the change:** We need to change the link to prod. We need to use the product name officially.
> 
> @dhaiducek and I are wondering how external users get to an internal channel?
> 
> **Motivation for the change:** Product guidelines and repo structure.

the doc link is addressed in PR #39 

If you want to use the "official" name then wouldn't that be `Red Hat Advanced Cluster Management for Kubernetes`?  This whole github org (github.com/open-cluster-management) is the upstream "project" called `Open Cluster Management` the downstream "product" is called `Red Hat Advanced Cluster Management for Kubernetes`... I think that using the upstream "project" name is fine in an asset that will never leave this github org.

**Comment by tpouyer** [comment]
@dhaiducek About the internal forum-acm slack channel link... it is a requirement of this early release that all users be RedHat employees or specifically allowed users who are members of our quay.io org in order to be able to download the images that this repo will attempt to install... I don't see a problem with the internal link here, for now, although I should probably put some context around that link that states it's for RedHat employee's only.

**Comment by dhaiducek** [comment]
@tpouyer Thanks for confirming--that was our concern. That the repo was externalized but there were no indications that these were internal-only instructions.

**Comment by bcpswope** [comment]
@tpouyer see my change: the `Open Cluster Management` organization

**Comment by bcpswope** [comment]
and updated the link to readme to prod

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 I'm really glad you're reading this, because we need your help to make this project come to fruition.
 
-If you haven't already, come find us on Slack ([#forum-acm](https://coreos.slack.com/archives/CTDEY6EEA)). We want to help answer your questions about Open Cluster Management.
+If you haven't already, come find us on Slack ([#forum-acm](https://coreos.slack.com/archives/CTDEY6EEA)). We want to help answer your questions about the `Open Cluster Management` organization.
```
Comment (tpouyer) [inline CONTRIBUTING.md:5]: @bcpswope Can you add a note here in the #forum-acm link that indicates this is an "internal RedHat" slack channel only.

**Pull request merged**